master/Makefile

```diff
@@ -1,7 +1,7 @@
 .DEFAULT_GOAL := build
 SHELL := bash
 GENERATED := packaging/LICENSE
-GENERATION_INPUTS = ../schemas/gen.py $(shell find ./pkg/schemas/ -name '*.go') $(shell find ../schemas/expconf -name '*.json')
+GENERATION_INPUTS = ../schemas/gen.py $(shell find ../schemas/expconf -name '*.go') $(shell find ../schemas/expconf -name '*.json')
```

Comment:
there are no `../schemas/expconf` files with `.go`. i'm trying to remember/understand why the stuff in `./pkg/schemas/*.go` was an input in the first place.

Reply:
from what I understand, the make target is running `$(GENERATION_INPUTS)` from inside `pkg/schemas` which is why the path `../schemas/expconf` works here to find all the generated `*.go` files. `clean` runs `ungen` which removes all `zgen_*.go` files and the Makefile with ` $(GENERATION_INPUTS)` as defined now doesn't provide the right path to generate the `zgen_*.go` files again.
```
build/schema_gen.stamp: $(GENERATION_INPUTS)
	go generate ./pkg/schemas/...
	mkdir -p build
	touch $@

ungen:
	rm -f $(GENERATED)
	rm -f `find ./pkg/schemas/ -name 'zgen_*.go'` build/schema_gen.stamp
	rm -f `find ./internal/mocks -name '*.go'` build/mock_gen.stamp

clean: ungen
	rm -rf dist
	rm -rf build
```